src/css/Parser.js

```diff
@@ -1986,6 +1986,11 @@ Parser.prototype = function() {
                     functionText = tokenStream.token().value;
                     this._readWhitespace();
                     expr = this._expr(true);
+                    if (expr === null) {
+                      // the FUNCTION is set if there's a ( . It doesn't check for a closing ) in identOrFunctionToken in TokenStream..
```

Comment:
This is a bit confusingly worded. Perhaps this:

"The FUNCTION token has already checked that there is an opening (."

Or omit this line of comment entirely. The level of commenting in the line below is normal for other similar situations in the codebase.